datadog_test.go

```diff
@@ -272,3 +272,48 @@ func TestHistogram(t *testing.T) {
 		t.Errorf("Expected: %v, Got: %v\n", vd, actual)
 	}
 }
+
+func TestPercentile_buildMetricNameForPercentile(t *testing.T) {
```

Comment:
Can we please not use underscores in test names? We usually don't. Maybe we can just `TestPercentileNames`?

Reply:
Moved to stats.go and renamed to TestPercentileName